ros_support_interfaces/dummy_pkg_for_declarations/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(dummy_pkg_for_declarations)
```

Comment:
Apologies for the drive by review, but I'd really consider calling this something more "metapackagey" -- like `"convince_ros_support_interfaces"`.

Then, you can add all the other packages in this folder, plus the upstream interfaces, as dependencies.

This will let you do e.g.

```
rosdep install --whatever --the --args --are
colcon build --packages-up-to convince_ros_support_interfaces
```

Reply:
Hi, and thanks for the hint!
I definitely agree with you that the "dummy" word in the package name could be improved, but I would leave the "convince" name out of it.
My proposal is `as2fm_test_ros_interfaces`, and all other packages defining interfaces (e.g. `delib_ws_24_interfaces`) should be added as a runtime dependency.